src/agent/rustjail/src/container.rs

```diff
@@ -336,7 +336,7 @@ pub fn init_child() {
         Ok(_) => (),
         Err(e) => {
             log_child!(cfd_log, "child exit: {:?}", e);
-            write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str());
+            let _ = write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str());
```

Comment:
The logic surrounding this looks backwards to me.   In the `Ok()` case we're falling through and doing a `std::process::exit(-1)`, but in the `Err()` case we `return` and so keep going.

Reply:
@dgibson we can replace `std::process::exit(-1)` with unreachable!()

Reply:
I will issue a PR to resolve these unreachable codes.

Reply:
https://github.com/kata-containers/kata-containers/pull/820